2.9.0 defaults.json missing

gsd
node:internal/modules/cjs/loader:1383
  const err = new Error(message);
              ^

Error: Cannot find module './defaults.json'
Require stack:
- /Users/user/.nvm/versions/node/v22.22.0/lib/node_modules/gsd-pi/node_modules/@gsd/pi-coding-agent/dist/core/lsp/config.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
    at Function._load (node:internal/modules/cjs/loader:1192:37)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
    at Module.require (node:internal/modules/cjs/loader:1463:12)
    at require (node:internal/modules/helpers:147:16)
    at file:///Users/user/.nvm/versions/node/v22.22.0/lib/node_modules/gsd-pi/node_modules/@gsd/pi-coding-agent/dist/core/lsp/config.js:11:18
    at ModuleJob.run (node:internal/modules/esm/module_job:343:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/user/.nvm/versions/node/v22.22.0/lib/node_modules/gsd-pi/node_modules/@gsd/pi-coding-agent/dist/core/lsp/config.js'
  ]
}

Node.js v22.22.0

--- a/packages/pi-coding-agent/package.json
+++ b/packages/pi-coding-agent/package.json
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "build": "tsc -p tsconfig.json && npm run copy-assets",
-    "copy-assets": "node -e \"const{mkdirSync,cpSync}=require('fs');mkdirSync('dist/modes/interactive/theme',{recursive:true});cpSync('src/modes/interactive/theme','dist/modes/interactive/theme',{recursive:true,filter:(s)=>!s.endsWith('.ts')});mkdirSync('dist/core/export-html/vendor',{recursive:true});cpSync('src/core/export-html/template.html','dist/core/export-html/template.html');cpSync('src/core/export-html/template.css','dist/core/export-html/template.css');cpSync('src/core/export-html/template.js','dist/core/export-html/template.js');cpSync('src/core/export-html/vendor','dist/core/export-html/vendor',{recursive:true,filter:(s)=>!s.endsWith('.ts')})\""
+    "copy-assets": "node -e \"const{mkdirSync,cpSync}=require('fs');mkdirSync('dist/modes/interactive/theme',{recursive:true});cpSync('src/modes/interactive/theme','dist/modes/interactive/theme',{recursive:true,filter:(s)=>!s.endsWith('.ts')});mkdirSync('dist/core/export-html/vendor',{recursive:true});cpSync('src/core/export-html/template.html','dist/core/export-html/template.html');cpSync('src/core/export-html/template.css','dist/core/export-html/template.css');cpSync('src/core/export-html/template.js','dist/core/export-html/template.js');cpSync('src/core/export-html/vendor','dist/core/export-html/vendor',{recursive:true,filter:(s)=>!s.endsWith('.ts')});mkdirSync('dist/core/lsp',{recursive:true});cpSync('src/core/lsp/defaults.json','dist/core/lsp/defaults.json')\""
   },
   "dependencies": {
     "@gsd/pi-agent-core": "*",

